src/jdk.compiler/share/classes/com/sun/tools/javac/main/Option.java

```diff
@@ -89,6 +89,11 @@
  * or deletion without notice.</b></p>
  */
 public enum Option {
+    /*
+     * Maintenance note: when adding new annotation processing related
+     * options, the list of options regarded as requesting explicit
+     * annotation processing in JavaCompiler should be updated.
+     */
```

Comment:
(minor) consider moving this option to before the declaration of `Option` so that it seems less like it applies to the first member.  It could even be an `@implNote` on the enum.